root/atp-lib.tsx

```diff
@@ -48,7 +66,7 @@ export default function AtpLib({
       toggleMode,
     }),
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [mode]
+    [mode, mergedTheme]
```

Comment:
if we dont add `mergedTheme` to the deps, what if `customTheme` changes, would it re-render the app?